yarn.lock

```diff
@@ -12374,7 +12374,7 @@ terser@^4.1.2, terser@^4.6.3:
     source-map-support "~0.5.12"
 
 terser@^5.0.0, terser@^5.3.4, terser@^5.7.2:
-  version "5.14.1"
+  version "5.14.2"
```

Comment:
@albinAppsmith : We should not change the yarn.lock file but should add resolution in package.json